bootstrap.php

```diff
@@ -23,7 +23,7 @@
  * @copyright Since 2007 PrestaShop SA and Contributors
  * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
  */
-$rootDir = getenv('_PS_ROOT_DIR_');
+$rootDir = _PS_ROOT_DIR_;
```

Comment:
To ensure that the constant is defined 
```suggestion
$rootDir = defined('_PS_ROOT_DIR_') ? _PS_ROOT_DIR_ : false;
```